lib/main.js

```diff
@@ -134,6 +138,11 @@ function run() {
             for (const tag of effectiveAdditionalTags) {
                 buildParams.push("-t", `${config.repository}:${tag}`);
             }
+            if (!config.buildArgs.length) {
```

Comment:
I'd use a `for (const arg of config.buildArgs)` here without testing for length like this, I think?